res/themes/light-high-contrast/css/_light-high-contrast.scss

```diff
@@ -129,3 +129,113 @@ $roomtopic-color: $secondary-content;
 .mx_Dialog_buttons button.mx_LocationPicker_cancelButton::before {
     background-color: $background !important;
 }
+
+.mx_SpotlightDialog_wrapper .mx_Dialog {
+    #mx_SpotlightDialog_keyboardPrompt {
+        > span > div {
+            color: $background !important;
+        }
+    }
+}
+
+.mx_SpotlightDialog {
+    .mx_SpotlightDialog_searchBox {
+        > .mx_SpotlightDialog_filter {
+            color: $background !important;
```

Comment:
given how many things got overriden with $background this feels like the wrong solution. I think we should introduce a new intermediary variable(s) which abstracts this away in the majority of cases and where sane. This many overrides are a bit hard to fathom